doc/rados/configuration/filestore-config-ref.rst

```diff
@@ -302,7 +302,7 @@ Misc
 
 ``filestore split multiple``
 
-:Description:  ``filestore_split_multiple * abs(filestore_merge_threshold) * 16`` 
+:Description:  ``(filestore_split_multiple * abs(filestore_merge_threshold) + (rand() % filestore_split_rand)) * 16``
```

Comment:
good catch. yeah, these aren't the most intuitive options, but at this point I'd like to change filestore as little as possible